### PR TITLE
add fastlane file for both CI and local

### PR DESCRIPTION
**Pull request by huminzhi**

### CODE REVIEW

File: fastlane/Fastfile [hunk 1]
```diff
@@ -0,0 +1,35 @@
+system("npm install shelljs@0.7.0")
+system("npm install -g cordova")
+system("cordova telemetry on")
```
Comment (bhariharan) [inline fastlane/Fastfile]: Should be `cordova telemetry off`.

File: fastlane/Fastfile [hunk 2]
```diff
@@ -0,0 +1,35 @@
+system("npm install shelljs@0.7.0")
+system("npm install -g cordova")
```
Comment (wmathurin) [inline fastlane/Fastfile]: Should be
```
system("npm install -g cordova@6.4.0")
```

File: fastlane/Fastfile [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+	if $jenkins.include?"Android-Package"
+		$os = "android"
+  elsif $jenkins.include?"iOS-Package"
+		$os = "ios"
+	end
+end
+
+lane :all do
+	system("set -o pipefail && node ../test/test_force.js --os=#$os --apptype=native,native_swift,react_native,hybrid_remote,hybrid_local")
+	system("set -o pipefail && node test_force.js --os=#$os --templaterepouri=https://github.com/forcedotcom/SmartSyncExplorerReactNative#master")
```
Comment (wmathurin) [inline fastlane/Fastfile]: Right now SmartSyncExplorerReactNative at master uses the unstable SDKs (see https://github.com/forcedotcom/SmartSyncExplorerReactNative/blob/master/package.json#L12)
When we release 5.0, we should change the package.json on master to point to v5.0.0 (and tag it), and have an unstable branch pointing to the unstable SDKs. At that point, this fastlane script should point to unstable (in the unstable branch of SalesforceMobileSDK-Package) and it should point to v5.0.0 on master.

Comment (wmathurin) [inline fastlane/Fastfile]: We should probably have a task in the release checklist to make sure all the pointers / tags are correct across all repos.

**Comment by wmathurin** [comment]
👍

**Pull request merged**